R/tic.R

```diff
@@ -13,8 +13,7 @@ use_tic <- function(path = usethis::proj_get(), quiet = FALSE) {
   #' The preparation consists of the following steps:
   withr::with_dir(path, {
     #' 1. If necessary, create a GitHub repository via [use_github()]
-    use_github_interactive()
-    stopifnot(uses_github())
+    # FIXME: DO this with the 'gh' pkg
```

Comment:
We now have a better version of `use_tic()` and `use_travis_deploy()` in {tic}. I still think `use_tic()` belongs in {tic}. Can we keep responsibility for defining environment variables in {tic} and make `use_travis_deploy()` accept arguments that are necessary for setting up deployment (public SSH key, `"id_rsa"` environment variable)?

Reply:
I would want to remove `R/tic.R` completely and have all of this in _tic_. I see no reason why this should live in _travis_.
There is the new `use_tic()` now in `tic()` and I would propose that _travis_ should only do things related to travis and only the minimal set of Github & other interactions.

In the same call I would remove `use_travis_deploy()` from _tic_ because this should live in _travis_.